projects/igniteui-angular/src/lib/core/styles/components/grid-toolbar/_grid-toolbar-component.scss

```diff
@@ -16,6 +16,7 @@
                 igx-column-actions,
                 igx-icon,
                 igx-progress-linear,
+                igx-overlay,
```

Comment:
I wonder if this should also be `igx-drop-down`? It seems to be working even without the drop-down, just the overlay, but semantically it uses a drop down component inside. Question remains why the drop down theme is not needed here.

Reply:
It most definitely is needed.